workflow/tests/ACCA_Examples/Walker_Residence.xml

```diff
@@ -72,65 +116,638 @@
         </AirInfiltration>
         <Roofs>
           <Roof>
-            <SystemIdentifier id='Roof1'/>
+            <SystemIdentifier id='RoofBedroom1'/>
+            <AttachedToSpace idref='bedroom_1'/>
             <InteriorAdjacentTo>attic - vented</InteriorAdjacentTo>
-            <Area>2015.0</Area>
```

Comment:
Roof area here I could not track where this came from, but I used the same areas from Smith since they're the same layout on the first floor

Reply:
The roof is over an unconditioned attic, and _it's the attic floor that is associated with different rooms_. So it's weird to have an attic roof surface associated with a space like a bedroom, it's not directly connected to it. I suggest going back to a single `Roof` element and removing the `AttachedToSpace`. (More generally, for any surface that is not adjacent to conditioned space, there's no point splitting it up into multiple surfaces, nor assigning it to a `Space`.)

Reply:
Sounds reasonable, I used to require space assignments for all surfaces, but no longer a requirement after relaxing the rules. So this is allowed and makes more sense, I'll change it.